index.html

```diff
@@ -38,6 +40,195 @@ <h1>
+                      <ul>
+                        <li class="option">Canopy</li>
+                        <li> <i class="fas fa-circle"></i> Back End Dev</li>
+                        <li><i class="fas fa-circle"></i> 2015</li>
+                      </ul>
+                      </div>
+                    <p>A daily selection of privately personalized reads, no accounts or sign-ups required. </p>
+                    <div class="languages inline">
+                      <ul>
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javascript</li>
+                      </ul>
+                    </div>
+                    <a href="#" class="btn-one">See project</a>
+                  </li>
+                </ul>
+              </div>
+           </li>
+            <li class="cards">            
+              <div class="cards-container">
+                <ul>
+                  <li>
+                      <img src="images/card4.png" alt="Forth Card">
+                  <li class="features">
+                    <h1><a class="head" href="#">Multi-Post Stories</a></h1>
+                    <div class="information">
+                      <ul>
+                        <li class="option">Canopy</li>
+                        <li> <i class="fas fa-circle"></i>Back End Dev</li>
+                        <li><i class="fas fa-circle"></i>2015</li>
+                      </ul>
+                    </div>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups requires</p>
+                    <div class="languages inline">
+                      <ul>
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javascript</li>
+                      </ul>
+                    </div>
+
+                    <a href="#" class="btn-one">See project</a>
+                  </li>
+                </ul>
+              </div>
+           </li>
+      </ul>
+</section>
+<section class="section-about">
+<div class="container">
+<div class="about-grid">
+    <div class="text">
+      <div class="headline-text">
+          <h1>About Myself</h1>
+
+            <p class="paragraph1">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a
+             project you need coded, don’t hestiate to contact me.</p>
+             <p class="lets-connect"> LET'S CONNECT</p>
+
+             <ul class="buttons">
+                 <li><a> <img src="./images/twitter.png" alt=""> </a></li>
+                 <li><a> <img src="./images/Linkedin.png" alt=""></a></li>
+                 <li><a> <img src="./images/medium.png" alt=""></a></li>
+                 <li><a> <img src="./images/github.png" alt=""></a></li>
+                 <li><a> <img src="./images/angellist.png" alt=""></a></li>
+             </ul>
```

Comment:
- [ ] For all of your link tags (`a` tag) you should add `href` attribute.
- [ ] If you don't have an available link for the `a` tag, just put `href` value as `"#"`. For instance:
```
<a href="#"><a>
```

Reply:
Thank you I really appreciate the elaboration and the explanation. I'll see to it that I make the appropriate corrections.